### ISSUE TITLE
this.once is not a function after browserify build of 4.0.0

### ISSUE THREAD

**Issue by cordovapolymer** [maxogden/websocket-stream#114]
this.once is not a function after browserify build of 4.0.0, it was ok in prior versions

**Comment by RangerMauve** [comment]
Was this a browserify build or a react-native build? What version of browserify are you using and what are the configuration options that were passed in?

**Comment by cordovapolymer** [comment]
@RangerMauve , I tried building with browserify 14.1.0 and 14.3.0, I'm using it with https://github.com/arian/partition-bundle, here's the build script:

```
var browserify = require('browserify');

var b = browserify(['./lib/load', './lib/other']);

b.transform({
  global: true
}, 'uglifyify');

var output = './scripts/';
b.plugin('partition-bundle', { map: './lib/bundle.json', output: output, main: './main' });

```

**Comment by mcollina** [comment]
I've released readable-stream 2.2.9. Can you please check if it solves the problem?

**Comment by cordovapolymer** [comment]
@mcollina , yes it's resolved thank you!
